src/index.js

```diff
@@ -4,7 +4,7 @@ import { Provider } from 'react-redux';
 import { createStore, applyMiddleware, compose } from 'redux';
 import thunk from 'redux-thunk';
 import rootReducer from './store/reducers';
-import App from './components/App';
+import App from './components/app';
```

Comment:
The filename in the import changed from upper to lowercase but the actual filename of the file was not updated. The file in the repo is still _src/components/App.js_ so that should be updated as well.

Reply:
This is weird, I made the changes to the file name and committed it previously. 
![Screen Shot 2019-04-02 at 4 13 04 PM](https://user-images.githubusercontent.com/14690224/55433002-5c95fc00-5562-11e9-9534-5b11a7ece8e5.png)

Reply:
NVM. Found out why it wasnt part of the previous commit. https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git